Dockerfile.data_cube_creation

```diff
@@ -11,9 +11,8 @@ ENV PIP_NO_CACHE_DIR=1
 
 WORKDIR /
 
-ADD backend/wmg/pipeline/requirements.txt requirements.txt
-ADD requirements.txt requirements-base.txt 
-RUN pip3 install -r requirements-base.txt -r requirements.txt 
+ADD requirements-wmg-pipeline.txt requirements-wmg-pipeline.txt
+RUN pip3 install -r requirements-wmg-pipeline.txt
```

Comment:
Out-of-scope, but it would be nice to align the names of Dockerfiles, requirements files, and python module names. If it doesn't cause downstream pain, could rename this to `Dockerfile.wmg-pipeline`.

Reply:
@atolopko-czi - will do